pages/common/qmv.md

```diff
@@ -0,0 +1,24 @@
+# qmv
+
+> Move files and directories with your favorite text editor.
+> More information: <https://www.nongnu.org/renameutils/>.
+
+- Move a single file (invoke an editor with source on the left, target on the right):
+
+`qmv {{source_file}}`
+
+- Move multiple JPG files:
+
+`qmv {{*.jpg}}`
+
+- Move directories:
+
+`qmv -d {{directory/}} {{another_directory/}} {{the_other_directory/}}`
```

Comment:
path/to/directory, path/to/another_directory, path/to/target_directory

Reply:
There are no target directory there, if the command get 3 arguments, then the 3 of them will be move (telling targets in the editor later).

But I get the idea, fix in bb2f8e0566cad4f47c037ca682480b136aa70ded.

Reply:
Ah, I think @agnivade is asking where do the directories get moved to, @neizod?